### DISCUSSION TITLE
最近停贷数量很少增加了，是真的控制住了吗？

### DISCUSSION THREAD

**Discussion by freedom1933** [WeNeedHome/SummaryOfLoanSuspension#970] [Q&A]
最近停贷数量很少增加了，是真的控制住了吗？

**Comment by Accelerator6666** [comment]
解决不了烂尾楼，还解决不了买了烂尾楼的人？

**Comment by xiaer93** [comment]
热度控制住了，毕竟国内舆论环境……

也没人敢在微信群中讨论，怕查水表和封群…

**Reply by jz03** [reply]
掩耳盗铃之举，不敢直接面对问题。

**Comment by jingzz1** [comment]
“烂尾”，“断供”已经是大数据关键词，知乎搜索“断供潮”查不到任何消息，b站有关键字也会被限流，断供声明发一个删一个。

**Reply by AI-Adventist** [reply]
搞不好这个数据库时不时也可能消失，不是有官方数据是烂尾多少平米么，干嘛防川

**Reply by John-zhan** [reply]
> “烂尾”，“断供”已经是大数据关键词，知乎搜索“断供潮”查不到任何消息，b站有关键字也会被限流，断供声明发一个删一个。

牛逼啊，浑身g点

**Comment by imhappyok** [comment]
319套3天没变化，看来已经没多大影响力了，压住了也就意味着这事可以不解决，无产阶级的第一次抗争失败了，老老实实还贷，别瞎想

**Reply by solidusxin** [reply]
就是控制住了发声的渠道，又不是那么多人懂得去海外平台发。再说，海外平台发，大概率抓进去发的人。不过我在想，是不是可以用邮件，邮箱发外媒他们控制不住，相当于公告了。不过业主可能也不懂这些。最多再工地贴告示啥的。网络上基本找不到任何信息

**Reply by chenjiasu** [reply]
又来劝投降了？

**Comment by zilazhu** [comment]
一群刁民汉奸，还跑到外网上来发东西，洋上访，告洋状？
农逼屁民也配说抗争，欠坦克压了。

**Comment by TALK1949** [comment]
与其说减少，不如说最初的消息源是如何获得这么多数据的，中国作为一个被打散的原子化社会，在各方面监控都无孔不入的情况下，业主们是如何做到短时间内动员起来共同停贷的，所以之前有人怀疑是派系斗争，或有内部人员参与，也不是不无道理。

**Comment by Zoey-nn** [comment]
是把烂尾楼业主控制住了 社区警察局没事就给你打电话让你别去闹 真的呵呵了

**Comment by ytx0574** [comment]
不, 很多群里还是在默默的号召停, 比如我就停了, 只有大面积停, 政府才会理你

**Comment by oceanzhang860411** [comment]
停貸就是風險，控制風險，你説控制的是什麽？自己去想
